snuba/datasets/events.py

```diff
@@ -8,11 +9,178 @@ class EventsDataSet(DataSet):
     and the particular quirks of storing and querying them.
     """
 
-    SCHEMA = TableSchema(
-        local_table_name='sentry_local',
-        dist_table_name='sentry_dist')
+    def __init__(self):
```

Comment:
This constructor never calls `super`. It don't think that has any side effects right now, but it probably should to avoid future confusion.

Reply:
will fix